ManatiCR/manatime
# 12: Time entries content type.

**Steps to test**
- Check if the date dependency was added automatically.
- Verify the manatime_feature_time_entries was created fine.
- Check if the human and machine names are right.
- Check the Time entries's fields.
- Review if manatime_feature_time_entries was added fine to settings.php.

[Issue link](https://github.com/ManatiCR/manatime/issues/12)
